guide.py

```diff
@@ -0,0 +1,434 @@
+      feature.shipped_milestone = shipped_milestone
+      feature.shipped_android_milestone = shipped_android_milestone
+      feature.shipped_ios_milestone = shipped_ios_milestone
+      feature.shipped_webview_milestone = shipped_webview_milestone
+      feature.shipped_opera_milestone = shipped_opera_milestone
+      feature.shipped_opera_android_milestone = shipped_opera_android_milestone
+      feature.footprint = int(self.request.get('footprint'))
+      feature.interop_compat_risks = self.request.get('interop_compat_risks')
+      feature.ergonomics_risks = self.request.get('ergonomics_risks')
+      feature.activation_risks = self.request.get('activation_risks')
+      feature.security_risks = self.request.get('security_risks')
+      feature.debuggability = self.request.get('debuggability')
+      feature.all_platforms = self.request.get('all_platforms') == 'on'
+      feature.all_platforms_descr = self.request.get('all_platforms_descr')
+      feature.wpt = self.request.get('wpt') == 'on'
+      feature.wpt_descr = self.request.get('wpt_descr')
+      feature.visibility = int(self.request.get('visibility'))
+      feature.ff_views = int(self.request.get('ff_views'))
+      feature.ff_views_link = ff_views_link
+      feature.ff_views_notes = self.request.get('ff_views_notes')
+      feature.ie_views = int(self.request.get('ie_views'))
+      feature.ie_views_link = ie_views_link
+      feature.ie_views_notes = self.request.get('ie_views_notes')
+      feature.safari_views = int(self.request.get('safari_views'))
+      feature.safari_views_link = safari_views_link
+      feature.safari_views_notes = self.request.get('safari_views_notes')
+      feature.web_dev_views = int(self.request.get('web_dev_views'))
+      feature.web_dev_views_link = web_dev_views_link
+      feature.web_dev_views_notes = self.request.get('web_dev_views_notes')
+      feature.prefixed = self.request.get('prefixed') == 'on'
+      feature.spec_link = spec_link
+      feature.tag_review = self.request.get('tag_review')
+      feature.standardization = int(self.request.get('standardization'))
+      feature.doc_links = doc_links
+      feature.sample_links = sample_links
+      feature.search_tags = search_tags
+      feature.comments = self.request.get('comments')
+      feature.experiment_goals = self.request.get('experiment_goals')
+      feature.experiment_timeline = self.request.get('experiment_timeline')
+      feature.experiment_risks = self.request.get('experiment_risks')
+      feature.experiment_extension_reason = self.request.get('experiment_extension_reason')
+      feature.ongoing_constraints = self.request.get('ongoing_constraints')
+    else:
+      # Check bug for existing blink component(s) used to label the bug. If
+      # found, use the first component name instead of the generic "Blink" name.
+      try:
+        blink_components = self.__get_blink_component_from_bug(blink_components, bug_url)
```

Comment:
We likely don't do it very often, but this looks like an expensive call (that could fail, since it depends on a network fetch).  Any need for caching?

Reply:
Yeah, it would be challenging to do this right and make it reliable.   This is existing code that I don't think has been working for awhile.  I don't know if users even complained that it stopped working.  So, let's try leaving it out for a while, we can always add it back in if users say it should be a priority.